src/main/java/com/dawn/jat/illuminati/comment/entity/CommentEntity.java

```diff
@@ -0,0 +1,4 @@
+package com.dawn.jat.illuminati.comment.entity;
+
+public class CommentEntity {
```

Comment:
this is just a suggestion, we dont need to add a Comment right now :d

Reply:
Removed